src/fastfield/mod.rs

```diff
@@ -12,8 +12,8 @@
 //!
 //!
 //! Fields have to be declared as `FAST` in the  schema.
-//! Currently only 64-bits integers (signed or unsigned) are
-//! supported.
+//! Currently supported fields are: u64, i64 and f64.
+//! u8 are supported for advanced usage.
```

Comment:
unsure.

Reply:
where do you get the u8 part from?

Reply:
This is a &[u8] bytes fast field, not u8. You can use it like this:

```
let bytes_field = schema_builder.add_bytes_field("bytes", FAST | INDEXED | STORED);
```
Instead of a number you can associate arbitrary bytes with a doc and retrieve them fast via the fast field.

Reply:
Thx for the explanations. So something like this : https://github.com/quickwit-oss/tantivy/pull/1301/commits/d36e0a954986c35565e545886dd500ae4d4d19cd ?